packages/sirius-web/backend/sirius-web-tests/src/main/java/org/eclipse/sirius/web/tests/services/representation/CreateRepresentationRunner.java

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.tests.services.representation;
+
+import org.eclipse.sirius.components.collaborative.dto.CreateRepresentationInput;
+import org.eclipse.sirius.components.graphql.tests.api.IGraphQLRequestor;
+import org.eclipse.sirius.components.graphql.tests.api.IMutationRunner;
+import org.springframework.stereotype.Service;
+
+import java.util.Objects;
+
+/**
+ * Used to add a page to a form description editor.
+ *
+ * @author mcharfadi
+ */
+@Service
+public class CreateRepresentationRunner implements IMutationRunner<CreateRepresentationInput> {
```

Comment:
- CreateRepresentationMutationRunner instead
- It already exists in `org.eclipse.sirius.components.graphql.tests.CreateRepresentationMutationRunner`